terraform/gitops/k8s-cluster-config/app-deploy.tf

```diff
@@ -200,6 +200,11 @@ locals {
   pm4ml_keycloak_realm_env_secret_map = { for key, pm4ml in local.pm4ml_var_map :
     "${var.pm4ml_oidc_client_secret_secret}-${key}" => var.pm4ml_oidc_client_secret_secret_key
```

Comment:
does this go away with ory @kalinkrustev?

Reply:
This is in an area that I do not know yet. I see it relates to the experience API, where it is passed.

Reply:
we should review that with @geka-evk